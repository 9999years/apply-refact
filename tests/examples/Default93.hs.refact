[("tests/examples/Default93.hs:1:7-26: Suggestion: Use void\nFound:\n  baz baz >> return ()\nPerhaps:\n  Control.Monad.void (baz baz)\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 7, endLine = 1, endCol = 27}, subts = [("a",SrcSpan {startLine = 1, startCol = 7, endLine = 1, endCol = 14})], orig = "Control.Monad.void (a)"}])]
