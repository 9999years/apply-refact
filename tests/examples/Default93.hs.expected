yes = Control.Monad.void (baz baz)
